docs: update user faq page (#536)
elastisys/compliantkubernetes: docs/user-guide/faq.md

Port 8000 is the only allowed port for OpenID callback URL and is needed by the `kubectl` OpenID plugin. If that port is used locally, then `kubectl` will try to bind to port 18000 which is not allowed due to security concerns. Make sure that nothing is running locally that is using port 8000.

## "Connection reset by peer" when port-forwarding to postgres?

You may have seen this error when port-forwarding to postgres:
```console
Forwarding from 127.0.0.1:5432 -> 5432
Forwarding from [::1]:5432 -> 5432
Handling connection for 5432
Handling connection for 5432
portforward.go:406] an error occurred forwarding 5432 -> 5432: error forwarding port 5432 to pod, uid : failed to execute portforward in network namespace "": read tcp4 127.0.0.1:5432->127.0.0.1:5432: read: connection reset by peer
portforward.go:234] lost connection to pod
```

> The only fix we know of is to use an older version of `kubectl` when making this request, specifically `v1.21.14` or lower.

Reviewer: Make it sound less definite
```suggestion
A workaround for the issue is to use an older version of `kubectl` when making this request, specifically `v1.21.14` or lower.
```

```console
Forwarding from 127.0.0.1:5432 -> 5432
Forwarding from [::1]:5432 -> 5432
Handling connection for 5432
Handling connection for 5432
portforward.go:406] an error occurred forwarding 5432 -> 5432: error forwarding port 5432 to pod, uid : failed to execute portforward in network namespace "": read tcp4 127.0.0.1:5432->127.0.0.1:5432: read: connection reset by peer
portforward.go:234] lost connection to pod
```

You have two options to resolve this issue:

1. Send a request to your administrator to disable TLS in the postgres cluster. Although it sounds "bad", it does not compromise security, since;

    - Traffic between kubectl and the Kubernetes API is encrypted.
    - In-cluster network is trusted.

2. A workaround for the issue is to use an older version of `kubectl` when making this request, specifically `v1.21.14` or lower.

    To avoid always using an old `kubectl` version, you can give the binary another name when downloading the `v1.21.14` version, e.g. `kubectl-1.21`. This way your normal `kubectl` binary can be kept up to date.

    Then use that specific binary when making the port-forward request:
    ```bash

>     kubectl-1.21 -n $NAMESPACE port-forward $USER_ACCESS 5432

Reviewer: ```suggestion
    kubectl-1.21 -n $NAMESPACE port-forward svc/$USER_ACCESS 5432
```
